dashboards-observability/public/components/visualizations/annotations/annotations.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { useState } from 'react';
+
+interface AnnotationProps {
+  showInputBox: boolean;
+  onTextChange: Function;
+  onAddAnnotation: Function;
+}
+
+export const Annotations: React.FC<AnnotationProps> = (props) => {
+  const handleAddAnnotation = () => {
+    props.onAddAnnotation();
+  };
+
+  const handleEditAnnotation = () => {};
```

Comment:
add comments for empty methods.

Reply:
Removed 2 methods added for future use..